x/claim/keeper/hooks.go

```diff
@@ -0,0 +1,94 @@
+package keeper
+
+import (
+	"github.com/c-osmosis/osmosis/x/claim/types"
+	gammtypes "github.com/c-osmosis/osmosis/x/gamm/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+)
+
+func (k Keeper) AfterAddLiquidity(ctx sdk.Context, sender sdk.AccAddress) {
+	if k.SetUserAction(ctx, sender, types.ActionAddLiquidity) {
```

Comment:
We should rename SetUserAction if its used like this. Perhaps we should call it `CheckAndSetUserAction`?

Reply:
Ok

Reply:
Renamed